client/src/components/Notices/Company.js

```diff
@@ -11,41 +12,62 @@ import './Company.css'
 
 type Props = {|
   item: Candidate,
-  toggledOn: boolean,
-  toggle: (e: Event) => void,
+  similarity: number,
+  showSupplierInfo: boolean,
+  showCustomerInfo: boolean,
+  toggleSupplier: () => void,
+  toggleCustomer: () => void,
 |}
 
-const _Company = ({item, toggledOn, toggle}: Props) => {
-  const showCompanyDetails = toggledOn && item.eid
+const _Company = ({
+  item,
+  similarity,
+  showSupplierInfo,
+  showCustomerInfo,
+  toggleSupplier,
+  toggleCustomer,
+}: Props) => {
+  similarity = Math.round(similarity * 100)
```

Comment:
Rewriting a parameter of a function isn't a very safe practice. Moreover, it's used only once here, so just remove this line and use that `round` directly in JSX.